content/technotes/2018-12-03-restez-release.md

```diff
@@ -0,0 +1,336 @@
+---
+slug: restez
+title: 'restez: Locally query GenBank'
+package_version: 1.0.0
+authors:
+  - name: Dom Bennett
+    url: https://github.com/dombennett
+date: '2018-12-03'
+categories: technote
+topicid: 1292
+tags:
+  - r
+  - community
+  - software
+  - review
+  - onboarding
+  - packages
+  - bioinformatics
+  - genbank
+  - dna
+  - sequences
+  - data-access
+  - data-munging
+  - molecular-biology
+  - joss
+---
+
+## What is `restez`?
+
+R packages for interacting with NCBI have, to-date, depended on [API](https://en.wikipedia.org/wiki/Application_programming_interface) query calls via [NCBI's Entrez](https://www.ncbi.nlm.nih.gov/search/).
+For computational analyses that require the automated look-up of reams of sequence data, piecemeal querying via bandwith-limited requests
+is evidently not ideal. These queries are not only slow, but they depend on network connections and the remote server's consistent
+behaviour. Additionally, users who make very large requests over extended periods of time run the risk of being blocked.
+
+`restez` attempts to make large queries of NCBI more efficient by allowing users to download whole sections of NCBI, create a local
+database from these downloaded files and then query this mini-GenBank version instead.
+This process is far more efficient as the downloaded files are compressed and users can limit the size of the database by only creating it
+from sequences of interest (limiting by taxonomic domain and/or sequence size).
+
+`restez` tries to be user-friendly: a database can be set-up in just a few function calls (set path, download and create),
+a database can be queried with a consistent set of functions (the `gb_*_get()` functions),
+the number of arguments per function is limited, and the package is designed to integrate with pre-exisiting R packages
+that interact with NCBI ([`rentrez`](https://github.com/ropensci/rentrez) and
+[`phylotaR`](https://github.com/ropensci/phylotaR))
+
+For more a detailed description and for tutorials of the package, please visit the
+[`restez` website](https://ropensci.github.io/restez/).
+
+![restez_outline](/img/blog-images/2018-12-03-restez/outline.png)
+*Figure 1. Diagrammatic outline of the `restez` functions and folder structure. Data is downloaded from NCBI into a file path
+set by the user. Raw downloads are stored in "downloads/" the generated database is stored in "sql_db/". This database can
+then be queried with a series of `gb_*_get()` functions as well as some additional wrappers.*
+
+## Installation
+
+`restez` (v1.0.0) is available from CRAN.
+
+```r
+install.packages('restez')
+```
+
+Alternatively, the latest development version can be downloaded from `restez`'s [GitHub page](https://github.com/ropensci/restez).
+
+```r
+devtools::install_github(repo = 'ropensci/restez')
+```
+
+## Usage
+
```

Comment:
Please give a couple of sentences outlining the example you're going to walk through. Indicate what you might do with a set of phage sequences. i.e. can you turn it into an interesting example that might interest readers?